internal/namespaces/instance/v1/custom_runs.go

```diff
@@ -162,3 +169,42 @@ func customInstanceImageListRun(ctx context.Context, argsI interface{}) (i inter
 
 	return customImages, nil
 }
+
+func customInstanceServerTypeListRun(originalRun core.CommandRunner) core.CommandRunner {
```

Comment:
Add a comment explaining *why* we need to transform the original response. 
If I understand correctly, something like : 
> The original run returns data formatted as a map
> By default we unmarshal map into a flat key/value list
> The view for the result of this command should be a table
> So we transform the map into a slice 

We should assess whether that's something that could happen with other commands.